src/virtio/sound.c

```diff
@@ -0,0 +1,852 @@
+#include "sound.h"
+#include "config.h"
+#include "microkit.h"
+#include "virq.h"
+#include "virtio/mmio.h"
+#include "virtio/virtq.h"
+#include <sddf/sound/queue.h>
+
+#define DEBUG_SOUND
```

Comment:
Should we comment this?

Reply:
Done in https://github.com/au-ts/libvmm/commit/2cb17ba06fc65cce1241ff3e4de821e6e85f2316